src/pages/api/getAllFellow.ts

```diff
@@ -0,0 +1,16 @@
+import { type NextApiRequest, type NextApiResponse } from "next";
+import { prisma } from "src/server/db";
+
+const getAllFellow = async (req: NextApiRequest, res: NextApiResponse) => {
+  const block: number|undefined = parseInt(req.query.block)
```

Comment:
req.query.block is undefined, will throw a ts error. you might wanna use lodash or something to fetch block first